features/users.feature

```diff
@@ -0,0 +1,665 @@
+      }]
+      """
+    And nothing should have been added or deleted
+
+
+
+  Scenario: An organization member should be able to retrieve a user by its email.
+    When borgana sends a GET request to /api/users?email=luke.skywalker@localhost.localdomain
+    Then the response code should be 200
+    And the response body should be the following JSON:
+      """
+      [{
+        "id": "@idOf: lskywalker",
+        "name": "lskywalker",
+        "technical": false,
+        "primaryEmailMd5": "@md5",
+        "active": false,
+        "roles": [],
+        "createdAt": "@iso8601"
+      }]
+      """
+    And nothing should have been added or deleted
+
+
+
+  Scenario: An organization member should be able to retrieve a user by its user name.
+    When borgana sends a GET request to /api/users?search=hsolo
+    Then the response code should be 200
+    And the response body should be the following JSON:
+      """
+      [{
+        "id": "@idOf: hsolo",
+        "name": "hsolo",
+        "technical": false,
+        "primaryEmailMd5": "@md5",
+        "active": true,
+        "roles": [],
+        "createdAt": "@iso8601"
+      }]
+      """
+    And nothing should have been added or deleted
+
+
+
+  Scenario: An organization member should be able to retrieve a user by its email through search.
+    When borgana sends a GET request to /api/users?search=han
+    Then the response code should be 200
+    And the response body should be the following JSON:
+      """
+      [{
+        "id": "@idOf: hsolo",
+        "name": "hsolo",
+        "technical": false,
+        "primaryEmailMd5": "@md5",
+        "active": true,
+        "roles": [],
+        "createdAt": "@iso8601"
+      }]
+      """
+    And nothing should have been added or deleted
+
+
+
+  Scenario: An organization member should be able to retrieve a user by its name through name.
+    When borgana sends a GET request to /api/users?name=hsolo
+    Then the response code should be 200
+    And the response body should be the following JSON:
+      """
+      [{
+        "id": "@idOf: hsolo",
+        "name": "hsolo",
+        "technical": false,
+        "primaryEmailMd5": "@md5",
+        "active": true,
+        "roles": [],
+        "createdAt": "@iso8601"
+      }]
+      """
+    And nothing should have been added or deleted
+
+
+
+  Scenario: An organization member should be able to retrieve inactive users.
+    When borgana sends a GET request to /api/users?active=false
+    Then the response code should be 200
+    And the response body should be the following JSON:
+      """
+      [{
+        "id": "@idOf: lskywalker",
+        "name": "lskywalker",
+        "technical": false,
+        "primaryEmailMd5": "@md5",
+        "active": false,
+        "roles": [],
+        "createdAt": "@iso8601"
+      }]
+      """
+    And nothing should have been added or deleted
+
+
+
+  Scenario: An organization member should be able to retrieve users filtered by organization.
```

Comment:
I think this test would be clearer if it read "An organization member should be able to retrieve the users in his organization".